firefox-ios/Storage/ThirdParty/SwiftData.swift

```diff
@@ -86,11 +86,11 @@ open class SwiftData {
     let schema: Schema
     let files: FileAccessor
 
-    static var EnableWAL = true
-    static var EnableForeignKeys = true
+    static let EnableForeignKeys = true
 
     /// Used to keep track of the corrupted databases we've logged.
-    static var corruptionLogsWritten = Set<String>()
+    // TODO: 
```

Comment:
```suggestion
    // TODO FXIOS-12603 This global property is not concurrency safe
```

Looks like we need another ticket here, I made one: https://mozilla-hub.atlassian.net/browse/FXIOS-12603